arm-insights/2015-04-01/swagger/insightsClient.json

```diff
@@ -0,0 +1,1123 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "InsightsClient",
+    "version": "2015-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/microsoft.insights/eventtypes/management/eventcategories": {
+      "get": {
+        "tags": [
+          "Event"
+        ],
+        "operationId": "EventCategories_List",
+        "description": "The list of event categories.",
+        "parameters": [
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "2015-04-01",
```

Comment:
We did not want the users to send us the api-version as parameters since they rarely know which version is the right one. We we hide it this way, the exposed api doesn't have an api-version parameter.